server/routes/checkAnswers.js

```diff
@@ -47,3 +150,64 @@ module.exports = function Index({ formService, authenticationMiddleware }) {
 
   return router
 }
+
+// utility function - convert to string where input MAY be an array
+
+const convertNestedArraysToString = allElements => {
+  if (Array.isArray(allElements)) {
+    let finalString = ''
+    allElements.forEach(element => {
+      finalString += `${element}, `
+    })
+    finalString = finalString.slice(0, -2)
+    return finalString
+  }
+  return allElements
+}
+
+// utility function - convert to string when input WILL be nested objects
+const convertArrayOfObjectsToString = (dataArray = []) => {
+  let finalString = ''
+  dataArray.forEach(element => {
+    finalString += `${element.name}, `
+  })
+  return finalString.slice(0, -2)
```

Comment:
think this could use array.join? and remove this method: 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join

Reply:
agreed. It is much more succinct. 
replaced with 
const convertArrayOfObjectsToString = (dataArray = []) => {
  return dataArray.map(element => element.name).join(', ')
}

Reply:
agreed. It is much more succinct.
replaced with
const convertArrayOfObjectsToString = (dataArray = []) => {
return dataArray.map(element => element.name).join(', ')
}